### ISSUE TITLE
Request for change of name of the "Dbutils-Pro" project

### ISSUE THREAD

**Issue by britter** [drinkjava2/dbutils-pro#1]
Dear Yong Zhu,

the Apache Commons Community is very happy to see your contributions to Apache Commons DbUtils. Have you considered bringing some of your improvements back to the projects? We have extensive documentation on how to become a contributor [1,2].

While we welcome contributions to our projects, we also have to make sure the Apache Software Foundation trademark policy is adhered to. For this reason we have to ask you to change the name of the Dbutils-Pro project in a way that confusion with Apache Commons DbUtils. is precluded. Further more we have to ask you to refer to Apache Commons DbUtils. by it’s full name only (not „Dbutils“ but „Apache Commons DbUtils“). Lastly the Apache License, Version 2.0 requires attribution of used code in NOTICE file (see Apache License, Version 2.0 section 4 Redistribution [3]).

Regards,
Benedikt Ritter, on behalf of the Apache Commons Community

[1] http://commons.apache.org/volunteering.html <http://commons.apache.org/volunteering.html>
[2] http://commons.apache.org/patches.html <http://commons.apache.org/patches.html>
[3] https://www.apache.org/licenses/LICENSE-2.0.html#redistribution <https://www.apache.org/licenses/LICENSE-2.0.html#redistribution>

**Comment by drinkjava2** [comment]
Oh, never thought of the naming issue, for next release will think about use another name.

**Comment by drinkjava2** [comment]
Oh, never thought of the naming issue, for next release will change to "jDbPro", hope this works. I'll try register Apache site later but not right now because I'm very busy now. Thanks

**Comment by britter** [comment]
Hello @drinkjava2,

the name "jDbPro" would be fine for us.

Thank you!

**Comment by jwcarman** [comment]
@drinkjava2 yeah, don't hesitate to bring your suggestions and improvements to the Apache Commons folks.  We're always looking for good people to lend a hand!

**Comment by britter** [comment]
@drinkjava2 friendly reminder. We are still waiting for the rename.
